src/main/java/service/TheFileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class TheFileReaderImpl implements TheFileReader {
+    private static final int TITLE = 0;
+
+    @Override
+    public List<String> readFromFile(String fileName) {
+        List<String> reader;
+        try {
+            reader = Files.readAllLines(Paths.get(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("no file Bro!", e);
```

Comment:
```suggestion
            throw new RuntimeException("no file Bro!", e);
```
maybe will be better to write something like  
("Can't read from file" + fileName, e)